src/main/java/seedu/letsgethired/storage/JsonAdaptedInternApplication.java

```diff
@@ -84,6 +81,8 @@ public InternApplication toModelType() throws IllegalValueException {
         }
         final Status modelStatus = new Status(status);
 
-        return new InternApplication(modelCompany, modelRole, modelCycle, modelStatus);
+        final Note modelNote = new Note("");
```

Comment:
I think there should be a `@JsonProperty("note")` in the constructor of this file? And also a `note` field as well?

Perhaps we could also try to follow the other fields to specify this, for example:
```suggestion
        if (note == null) {
            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT, Note.class.getSimpleName()));
        }
        if (!Note.isValidNote(note)) {
            throw new IllegalValueException(Note.MESSAGE_CONSTRAINTS);
        }
        final Note modelNote = new Note(note);
```